Sources/SwiftFormat/PrettyPrint/TokenStreamCreator.swift

```diff
@@ -3386,12 +3386,31 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
   private func appendToken(_ token: Token) {
     if let last = tokens.last {
       switch (last, token) {
-      case (.comment(let c1, _), .comment(let c2, _))
-      where c1.kind == .docLine && c2.kind == .docLine:
-        var newComment = c1
-        newComment.addText(c2.text)
-        tokens[tokens.count - 1] = .comment(newComment, wasEndOfLine: false)
-        return
+      case (.break(.same, _, .soft(let count, _)), .comment(let c2, _))
```

Comment:
Since you're not using `count` anywhere else in the body of the statement, it would be simpler to put it directly in the pattern instead of in the `where` clause:

```swift
case (.break(.same, _, .soft(1, _)), .comment(let c2, _))
where c2.kind == .docLine || c2.kind == .line:
```